src/anet.c

```diff
@@ -479,16 +479,23 @@ int anetUnixServer(char *err, char *path, mode_t perm, int backlog)
     int s;
     struct sockaddr_un sa;
 
-    if ((s = anetCreateSocket(err,AF_LOCAL)) == ANET_ERR)
+    if (strlen(path) > sizeof(sa.sun_path)-1) {
+        anetSetError(err,"socket path too long - %s", path);
+        return ANET_ERR;
+    }
+    if ((s = anetCreateSocket(err,AF_LOCAL)) == ANET_ERR) {
         return ANET_ERR;
+    }
 
     memset(&sa,0,sizeof(sa));
     sa.sun_family = AF_LOCAL;
     strncpy(sa.sun_path,path,sizeof(sa.sun_path)-1);
-    if (anetListen(err,s,(struct sockaddr*)&sa,sizeof(sa),backlog) == ANET_ERR)
+    if (anetListen(err,s,(struct sockaddr*)&sa,sizeof(sa),backlog) == ANET_ERR) {
         return ANET_ERR;
```

Comment:
No need to add all these `{}` blocks. This just clutters the blame log.

Reply:
ACK - fixed in the last commit